packages/app-store/applecalendar/api/add.ts

```diff
@@ -39,6 +39,11 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       });
     } catch (reason) {
       logger.error("Could not add this caldav account", reason);
+
+      if (reason instanceof Error && reason.message === "Invalid credentials") {
+        return res.status(403).json({ message: reason.message });
```

Comment:
why 403? for others you used 400?

Reply:
nice catch :) thanks, yeah it should be 400.